src/Umbraco.Web.BackOffice/Controllers/MemberTypeController.cs

```diff
@@ -172,6 +173,23 @@ public IActionResult DeleteById(int id)
         return Ok(result);
     }
 
+    /// <summary>
+    ///     Returns where a particular composition has been used
+    ///     This has been wrapped in a dto instead of simple parameters to support having multiple parameters in post request
+    ///     body
+    /// </summary>
+    /// <param name="filter"></param>
+    /// <returns></returns>
+    [HttpPost]
```

Comment:
As above, this is a `Get` request, but you added a `Post` attribute to it, so we're now in an ambiguous state, this looks like a pure `Get` request to me, so no need to force it to be a  `Post`.